benchmarks/sha3.cc

```diff
@@ -1,9 +1,10 @@
 #include "util.h"
 
 #include "Hacl_SHA3.h"
-
 #include "Hacl_Streaming_SHA3.h"
 
+#include <openssl/evp.h>
```

Comment:
Isn't this in `util.h` already?

Reply:
I think I would leave this here, even though it is in `util.h`. Maybe even remove from `util.h`? Makes the benchmarks more self-contained. Otherwise it could get confusing when decide that we don't need `util.h` in a benchmark, remove the import, and the benchmarks don't work anymore.